client/app/scripts/superdesk-packaging/views/add-to-package.html

```diff
@@ -0,0 +1 @@
+<div sd-add-package-dropdown></div>
```

Comment:
It's maybe better to use `sdAddPackageDropdown` as an element instead of as an attribute - it makes it more obvious that the directive results in a new widget and doesn't just add some extra behavior to a `<div>` element (that's a recommended patterns for directives in general).
```html
<sd-add-package-dropdown />
```
On the other hand, we use attribute-level directives throughout the projects, even when those directives, in fact, create custom UI elements / widgets. If consistency is deemed more important, we can nevertheless leave this as is. 

Just an idea...